api/urls.py

```diff
@@ -1,5 +1,9 @@
 from django.urls import path
+from .views import lyric_detail
 
 from .views import test
 
-urlpatterns = [path("/test/", test, name="test")]
+urlpatterns = [
+    path("/test/", test, name="test"),
+    path("/api/lyric/<int:id>/", views.lyric_detail),

```

Comment:
you should change this to `/lyric/<int:id>/`

Reply:
resolved?

Reply:
Yeah